BOHO staging TRV10 logs

Ikshan-Tango opened the pull request.

sourabhsaini11 commented:
@Ikshan-Tango, please provide logs from preprod env so that PR can be considered for log approval

Ikshan-Tango commented:
@sourabhsaini11 isn't log approval required for transition as a NP from staging to pre-prod? We've TRV11 in preprod.

sourabhsaini11 commented:
No, it's for transitioning to prod from pre-prod @Ikshan-Tango

Ikshan-Tango commented:
Cool. Thanks

Ikshan-Tango commented:
@sourabhsaini11 raised Pre-Prod environment logs, added pramaan flows separately as well.

Ikshan-Tango commented:
@sourabhsaini11 can you please approve the PR or let us know what are the next steps?

Ikshan-Tango commented:
@sourabhsaini11 any updates ?

aagmanbhatt commented:
@sourabhsaini11 @92shreyansh , awaiting reviews here.

sourabhsaini11 commented:
@Ikshan-Tango
init
- if collector is BAP then params shouldn't be present at 
- For tag SETTLEMENT_WINDOW, correct format is P1D
- missing type at fulfillments

on_confirm
- fulfillments.authorization should only be present if ride is assigned

Flow-Successfull-Ride-Completion
Missing on_status with ride status completed & payment marked as PAID


Please submit flows for IGM & RSF as well

Ikshan-Tango commented:
@sourabhsaini11 These pointed out issues are for what flows? 

I will attach the IGM flows separately in the next commit. 

Regarding RSF, we've already submitted that and got it approved for TRV11 domain. You can find them here - "[https://github.com/ONDC-Official/verification-logs/tree/main/TRV11/Boho/RSF](https://github.com/ONDC-Official/verification-logs/tree/main/TRV11/Boho/RSF)". Correct me if i am wrong but do we need to do it again for TRV10? As far as I know, I don't think so its necessary after seeing other NP's logs.

Ikshan-Tango commented:
@sourabhsaini11 

Submitted the new iterations of the log flows, please note the following :-

> init
> 
> if collector is BAP then params shouldn't be present at payments
> For tag SETTLEMENT_WINDOW, correct format is P1D
> 

**FIXED: IN THE LATEST COMMIT**

> on_confirm
> 
> fulfillments.authorization should only be present if ride is assigned
> 

**UNFIXED: THIS IS SOMETHING OUT OF OUR HAND AS THE SELLER APP SENDS US ON_CONFIRM**

> Flow-Successfull-Ride-Completion
> Missing on_status with ride status completed & payment marked as PAID
> 

**FIXED: ADDED ALL STATUS AND ON_STATUS CALLS NOW THAT ARE INVOLVED IN THIS FLOW IN THE LATEST COMMIT**

> Please submit flows for IGM & RSF as well

**FIXED: ADDED IGM FLOW SEPARATELY AND REGARDING RSF : We've already submitted that and got it approved for TRV11 domain. You can find them here - "https://github.com/ONDC-Official/verification-logs/tree/main/TRV11/Boho/RSF". Correct me if i am wrong but do we need to do it again for TRV10? As far as I know, I don't think so its necessary after seeing other NP's logs**

cc @aagmanbhatt @92shreyansh

aagmanbhatt commented:
@sourabhsaini11 awaiting your review here

sourabhsaini11 commented:
@Ikshan-Tango
## confirm
tags **SETTLEMENT_WINDOW**, **SETTLEMENT_AMOUNT**, **DELAY_INTEREST** value mismatch b/w current & past calls
incorrect value found for tag **SETTLEMENT_AMOUNT**
**fulfullment.type** is sent additionally

## on_init
incorrect value found for tag **SETTLEMENT_AMOUNT**

## on_confirm
incorrect value found for tag **SETTLEMENT_AMOUNT**

amitsinha07 commented:
@Ikshan-Tango

# Feedbacks for IGM

- Kindly separate IGM logs from transactional logs — it would be helpful for me to review them.

JyotiSharma0711 commented:
Reminder:

Kindly note that this PR is scheduled to be closed within the next 1 to 2 weeks. Please ensure that any required changes from your side are addressed within this timeframe.

JyotiSharma0711 commented:
Please ensure all necessary updates from your end are completed.

This is to inform you that the pull request will be **closed by Monday**, as per the planned schedule.

If any changes are pending, kindly address them before the deadline.

Thank you for your cooperation.
